File: Project_Sourcecode_go/output/strategy/storeData/StoreData1.go
package storeData

import "cs586_project/datastore"

type StoreData1 struct {
}

func (s *StoreData1) StoreData(ds interface{}) {
	d := ds.(*datastore.DS1)
	d.Id = d.Temp_y
	d.Pin = d.Temp_p
	d.Balance = d.Temp_a
}

File: Project_Sourcecode_java/src/main/java/edu/iit/cs586/project/output/strategy/StoreData/StoreData1.java
package edu.iit.cs586.project.output.strategy.StoreData;

import edu.iit.cs586.project.datastore.DS1;
import edu.iit.cs586.project.datastore.DataStore;

/**
 * @author CharmingLee <charminglee@jieztech.com>
 * @date 2022/5/2 22:52
 */
public class StoreData1 implements StoreData {
    @Override
    public void storeData(DataStore ds) {
        ds.id = ds.temp_y;
        ds.pn = ds.temp_p;
        ((DS1) ds).b = ((DS1) ds).temp_a;
    }
}
